R/g.inspectfile.R

```diff
@@ -41,13 +45,13 @@ g.inspectfile = function(datafile, desiredtz = "", params_rawdata = c(),
             },
             "csv" = { dformat = FORMAT$CSV
             testheader = read.csv(datafile, nrow = 1, skip = 0, header = FALSE)
-            
```

Comment:
I was hoping that indentation of empty lines is standardised in RStudio regardless of how it is configured.
If you are working in RStudio could you try Ctrl+A Ctrl+I to see whether the indentation of empty lines follows the indentation of the surrounding code?

To me that would seem the most intuitive approach to indentation because it makes it easier to insert new code.

Reply:
I ran this for indentation

Reply:
Ok, no worries. When I do that command in RStudio, it changes all indentations back to how I normally have it. But anyway, I will can correct the indentation after the PR is merged as probably you have a different RStudio configuration.

Reply:
I figured out what is causing this:

In both RStudio global and project options there is the option for saving to 
![image](https://github.com/wadpac/GGIR/assets/8362333/1c9d8f37-8bc3-46d0-9f86-6e56ce7ff067)

It seems like you have this ticked (default I suppose) while I do not.